src/selectedtests/task_mappings/task_mappings_cli.py

```diff
@@ -113,10 +102,10 @@ def create(
 
     LOGGER.info(f"Creating task mappings for {evergreen_project}")
 
-    mappings = TaskMappings.create_task_mappings(
+    mappings, most_recent_version_analyzed = TaskMappings.create_task_mappings(
```

Comment:
should this be `mappings, _ = TaskMappings.create_task_mappings(` since you are discarding the version?

Reply:
👍  updated